modules/tinymce/src/plugins/table/main/ts/api/Commands.ts

```diff
@@ -118,6 +119,26 @@ const registerCommands = (editor: Editor, actions: TableActions, cellSelection:
     });
   };
 
+  const toggleRowHeader = () => {
+    getSelectionStartCell(editor).each((startCell) => {
```

Comment:
This should be using the `performActionOnSelection` helper function instead of implementing duplicate code.